src/resolvers/ResolverBuilder.ts

```diff
@@ -49,12 +50,13 @@ export class ResolverBuilder {
     return {
       fieldName: fieldName,
       typeName: gqlType.name,
-      schemaDefinition: `${fieldName}(id: ID!, input: ${gqlType.name}Input!): ${gqlType.name}!`,
+      schemaDefinition: `${fieldName}(id: ID!, input: ${gqlType.name}Input!): ${gqlType.name}!`,  
       action: ResolverType.UPDATE,
```

Comment:
So this was for historical reasons as we needed type to be displayed in the UI that is not longer relevant. We do not need all of those times. I think we might just need implementation. 
If you find any place here that requires action. Having typeName seems ok but rest of the fields are just legacy/not neeed.

![image](https://user-images.githubusercontent.com/981838/58962097-5a3e6100-87a2-11e9-87c7-d91588aefab8.png)

Reply:
Just fieldname, schemaDefinition and implementation is required as of now. Can merge fieldName and implementation into one.

Reply:
hmm.. me thinking - we can still keep field name
but implementation will contain full resolver including field name